independent-projects/tools/codestarts/README.adoc

```diff
@@ -8,6 +8,8 @@ This guide explains how to create and configure a Quarkus Codestart for an exten
 Codestarts provide a personalized Quarkus getting started experience and really show the Quarkus breadth.
 A Quarkus extension are able to provide one or more well defined codestarts which will contain the necessary resources and code examples required to get started using that particular extension.
 
+See the staging environment to get an understanding what new quickstarts are: https://stage.code.quarkus.io/[https://stage.code.quarkus.io/]
```

Comment:
You can point to code.quarkus.io directly in the doc, I will promote staging to production later today..